Fixed #27679 -- Doc'd that empty formsets display extra + min_num forms. (#13313)
django/django: docs/topics/forms/formsets.txt

    >>> formset.non_form_errors()
    ['Please submit 3 or more forms.']

.. note::

>     When a formset contains no initial data, the number of forms displayed is
>     the sum of ``min_num`` and ``extra``.

Reviewer: Maybe:
```suggestion
    Regardless of ``validate_min``, if a formset contains no data, then
    ``extra + min_num`` empty forms will be displayed.
```

Author: I've spent some time thinking about your suggestion. In short, I can't think of anything better. 

I didn't originally include a note about `validate_min` this is about `min_num`; however, as it's in a section about validation I think the clarification is helpful. 

I then thought about how to reword it, but the options I thought of resulted in another full sentence to explain the `validate_min` point which you have refined to just two words.
